test/integration/authorization_test.go

```diff
@@ -318,7 +319,7 @@ func TestAuthorizationResolution(t *testing.T) {
 	roleWithGroup.Name = "with-group"
 	roleWithGroup.Rules = append(roleWithGroup.Rules, authorizationapi.PolicyRule{
 		Verbs:     sets.NewString("list"),
-		Resources: sets.NewString("builds"), // TODO we may need to track these down
+		Resources: sets.NewString("builds"),
```

Comment:
@deads2k do we care that this used to be `Resources: sets.NewString("resourcegroup:builds")`?

Reply:
> @deads2k do we care that this used to be Resources: sets.NewString("resourcegroup:builds")?

For the test?  No.